static-libraries/js/khan-lite.js

```diff
@@ -251,6 +251,11 @@ function clear_messages(msg_type) {
     return $("#message_container");
 }
 
+// To clear all messages including django and ajax.
+function clearAllMessages(msg_type) {
```

Comment:
What is the purpose of the argument here, and what does this function do that clear_messages does not? If messages are appearing outside of #message_container that is a problem that should be addressed, not the addition of an almost identical function to the one directly above here.